crates/flycheck/src/json_workspace.rs

```diff
@@ -0,0 +1,102 @@
+//! A `cargo-metadata`-equivalent for non-Cargo build systems.
```

Comment:
Is there any reason for this being here? This might as well live in the main rust-analyzer crate

Reply:
Ah I see this deopends on the command handle stuff ... we should really rename this crate then